timemachine/potentials/bonded.py

```diff
@@ -244,3 +245,17 @@ def log_flat_bottom_bond(conf, params, box, bond_idxs, beta):
     # note the extra 1/beta is to be consistent with other potentials
     # so that energies have units of kJ/mol
     return jnp.sum(log_nrgs) / beta
+
+
+def harmonic_positional_restraint(x_init: Array, x_new: Array, box: Array, k: float = 4_000) -> Array:
```

Comment:
optional: `4_000` -> `DEFAULT_POSITIONAL_RESTRAINT_K`

Reply:
Done in https://github.com/proteneer/timemachine/pull/1393/commits/7388c815cf171e3cb01b3528215b7ed4b5a4c537

Also bumps the value back to 4000.0 to resolve an issue discussed offline.